lab-tracey-radcliffe/README.me

```diff
@@ -0,0 +1,4 @@
+#Instructions
+
+-Clone the data onto your local computer
+-Inside of the root directory of my lab-tracey-radcliffe file, run mocha to see the test
```

Comment:
Thanks for providing a readme! Since you have a bin, you should also provide instructions for how to use your CLI.